src/webgl/p5.Texture.js

```diff
@@ -252,4 +253,112 @@ p5.Texture.prototype.unbindTexture = function() {
   gl.bindTexture(this.glTarget, null);
 };
 
+/**
+ * Sets how a texture is be interpolated when upscaled or downscaled.
+ * Nearest filtering uses nearest neighbor scaling when interpolating
+ * Linear filtering uses WebGL's linear scaling when interpolating
+ * @method setInterpolation
+ * @param {String} downScale Specifies the texture filtering when
+ *                           textures are shrunk. Options are LINEAR or NEAREST
+ * @param {String} upScale Specifies the texture filtering when
+ *                         textures are magnified. Options are LINEAR or NEAREST
+ * @todo implement mipmapping filters
+ */
+p5.Texture.prototype.setInterpolation = function(downScale, upScale) {
+  var gl = this._renderer.GL;
+
+  if (downScale === constants.NEAREST) {
+    this.glMinFilter = gl.NEAREST;
+  } else if (downScale === constants.LINEAR) {
+    this.glMinFilter = gl.LINEAR;
+  } else {
+    // falling back to default
+    this.glMinFilter = gl.LINEAR;
```

Comment:
Is the format of this block in anticipation of adding more possible scaling constants in the future? both of the last two conditions result in gl.LINEAR (and same is done below).

Reply:
I think I did this on the off chance that someone mistypes one of the constants so that it would still just choose something that worked. Is it worth throwing an error there instead or at least a warning that the constant was not recognized and LINEAR was chosen instead?

Reply:
ah oops I think I wasn't clear! any reason not to just have the else part (and lose the else-if)?

Reply:
No reason at all except my oversight. Lets axe it!